app/models/maintenance_tasks/run.rb

```diff
@@ -188,11 +188,21 @@ def stuck?
     def csv_attachment_presence
       if Task.named(task_name) < CsvCollection && !csv_file.attached?
         errors.add(:csv_file, 'must be attached to CSV Task.')
-      elsif !(Task.named(task_name) < CsvCollection) && csv_file.attached?
+      elsif !(Task.named(task_name) < CsvCollection) && csv_file&.attached?
```

Comment:
Since `ActiveStorage::Attached::One` implements `blank?`, we may be able to use `present?` instead. But the safe operator is fine otherwise.

Reply:
Sure, let's change it!